pkg/reconciler/instances/connectivityproxy/connectivityproxy.go

```diff
@@ -0,0 +1,33 @@
+package connectivityproxy
```

Comment:
JFYI - as we discussed in our previous call, I created a bug for the issue with hyphens in pkg name: https://github.com/kyma-incubator/reconciler/issues/126